### PR TITLE
Improved Messages

### LINKED ISSUE #97: improve messages

### PR DESCRIPTION
**Pull request by GoraJakub**

### CODE REVIEW

File: src/app/game.ts [hunk 1]
```diff
@@ -203,40 +207,45 @@ export class Game {
...
 
-    this.addGameMessage(
-      `The miners did their best. You have got ${gold} gold ${
+    this.addGameMessage({
+      message: `The miners did their best. You have got ${gold} gold ${
         gold === 1 ? 'bar' : 'bars'
-      }.`
-    )
+      }.`,
+      type: MessageType.SUCCESS,
```
Comment (bartoszbialecki) [inline src/app/game.ts]: I wonder what message should be here, i think the most messages should be info, success I would make only something like winning the battle or in the future when we add functionality to add by random resources, then we can show it as a success.

File: src/app/game.ts [hunk 2]
```diff
@@ -203,40 +207,45 @@ export class Game {
...
 
-    this.addGameMessage(
-      `The sawmill is going strong. You have got ${wood}  ${
+    this.addGameMessage({
+      message: `The sawmill is going strong. You have got ${wood}  ${
         wood === 1 ? 'log' : 'logs'
-      } of wood.`
-    )
+      } of wood.`,
+      type: MessageType.SUCCESS,
```
Comment (bartoszbialecki) [inline src/app/game.ts]: Here I would make INFO

File: src/app/game.ts [hunk 3]
```diff
@@ -390,25 +399,28 @@ export class Game {
...
     this.population += house.population
   }
 
   private handlewWallWasBuilt(wall: Wall) {
-    this.addGameMessage(
-      `The new wall added ${wall.getDefence()} points to your defence.`
-    )
+    this.addGameMessage({
+      message: `The new wall added ${wall.getDefence()} points to your defence.`,
+      type: MessageType.WARNING,
```
Comment (bartoszbialecki) [inline src/app/game.ts]: Here, INFO, WARNING I would use only for e.g. when attack is close

File: src/app/game.ts [hunk 4]
```diff
@@ -390,25 +399,28 @@ export class Game {
...
     this.villageDefence += wall.getDefence()
   }
 
   private handleWarehouseWasBuilt(warehouse: Warehouse) {
-    this.addGameMessage(
-      `The capacity of the resource locker has increased. You can now store ${this.storageCapacity} resources.`
-    )
+    this.addGameMessage({
+      message: `The capacity of the resource locker has increased. You can now store ${this.storageCapacity} resources.`,
+      type: MessageType.WARNING,
```
Comment (bartoszbialecki) [inline src/app/game.ts]: INFO

File: src/app/models/buildings/building.ts [hunk 5]
```diff
@@ -88,9 +90,10 @@ export class Building {
         this.handleBuildingWasBuilt()
         this.game.handleBuildingWasBuilt(this)
 
-        this.game.addGameMessage(
-          `The ${this.getTitle()} was upgraded to the level ${this.getLevel()}.`
-        )
+        this.game.addGameMessage({
+          message: `The ${this.getTitle()} was upgraded to the level ${this.getLevel()}.`,
+          type: MessageType.SUCCESS,
```
Comment (bartoszbialecki) [inline src/app/models/buildings/building.ts]: I would make it INFO

File: src/assets/css/screens/gameScreen.css [hunk 6]
```diff
@@ -141,6 +141,24 @@
...
+}
+.game-messages__item--info {
+  color: white;
+}
+.game-messages__item--warning {
+  color: #ffcc00;
+  font-weight: 600;
+}
+.game-messages__item--error {
+  color: #cc0000;
```
Comment (bartoszbialecki) [inline src/assets/css/screens/gameScreen.css]: The contrast of this color is not so good. Try maybe #ff6666

**Review by pdybowski — APPROVED** [review]

**Review by bartoszbialecki — APPROVED** [review]

**Pull request merged**